nuxeo-core-binarymanager-s3-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-s3-direct-upload/nuxeo-s3-direct-upload.html

```diff
@@ -127,11 +128,26 @@
       }.bind(this));
     };
 
+    S3Provider.prototype.hasAbort = function() {
+      return true;
+    };
+
     S3Provider.prototype.hasProgress = function() {
       return true;
     };
 
+    S3Provider.prototype.abort = function(file) {
```

Comment:
Similar to the above comment, we aren't making use of this method anywhere else, so I wouldn't make it public.